README.md

```diff
@@ -1,6 +1,22 @@
-Running steamlink on a raspberrypi using [balena.io](https://www.balena.io/)
+# balena steamlink 
+
+Run steamlink on a Raspberry Pi using [balena.io](https://www.balena.io/)
+
+## Getting started 
+
+You can one-click-deploy this project to balena using the button below:
+
+[![deploy button](https://balena.io/deploy.svg)](https://dashboard.balena-cloud.com/deploy?repoUrl=https://github.com/Page-/steamlink&defaultDeviceType=raspberrypi4-64)
+
+## Manual Deployment
+
+Alternatively, deployment can be carried out by manually creating a [balenaCloud account](https://dashboard.balena-cloud.com) and application, flashing a device, downloading the project and pushing it via either Git or the [balena CLI](https://github.com/balena-io/balena-cli).
+
+
+## Device Variables
+
+### Environment Variables
```

Comment:
These are actually config variables rather than environment variables, there is one environment variable that can be used which is `DEBUG` and can be set to 1 to stop the container exiting if steamlink / x fails for debugging purposes